autopark/Vehicle.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace autopark
+{
+    enum Color
+    {
+        Blue = 1,
+        White,
+        Green,
+        Gray,
+        Yellow,
+        Red
+    }
+    class Vehicle : IComparable<Vehicle>
+    {
+        public string VehicleType { get; }
```

Comment:
Use class from VehicleType from Level 1 instead of string.

Reply:
Done